tests/scripts/hpo/hpo_sanity_tests.sh

```diff
@@ -173,12 +173,15 @@ function hpo_sanity_test() {
 
 		get_trial_json=$(${curl} ''${hpo_url}'?experiment_name=petclinic-sample-2-75884c5549-npvgd&trial_number='${i}'' -w '\n%{http_code}' 2>&1)
 
-		get_trial_json_cmd="${curl} ${url}?experiment_name="petclinic-sample-2-75884c5549-npvgd"&trial_number=${i} -w '\n%{http_code}'"
+		get_trial_json_cmd="${curl} ${hpo_url}?experiment_name="petclinic-sample-2-75884c5549-npvgd"&trial_number=${i} -w '\n%{http_code}'"
 		echo "command used to query the experiment_trial API = ${get_trial_json_cmd}" | tee -a ${LOG}
 
 		http_code=$(tail -n1 <<< "${get_trial_json}")
 		response=$(echo -e "${get_trial_json}" | tail -2 | head -1)
-		response=$(echo ${response} | cut -c 4-)
+
+		if [ ${response::3} == "000" ]; then
+			response=$(echo ${response} | cut -c 4-)
+		fi
```

Comment:
Can we add some comments on why this code block is needed?

Reply:
Added now